packages/mockyeah-cli/bin/mockyeah-record.js

```diff
@@ -45,7 +45,10 @@ const withName = (env, name, options = {}) => {
           request.get(`${adminUrl}/record-stop`, () => {});
         } else {
           // eslint-disable-next-line global-require, import/no-dynamic-require
-          require(env.modulePath).recordStop();
+          require(env.modulePath).recordStop(err => {
+            if (err) console.error(err);
+            process.exit(err ? 1 : 0);
```

Comment:
A comment wouldn't hurt to describe when you might get into an error state. I expect it's when attempting to stop after a remote mockyeah is prematurely shuts down.